internal/manifest/oci_index.go

```diff
@@ -84,6 +84,45 @@ func (index *OCI1IndexPublic) UpdateInstances(updates []ListUpdate) error {
 	return nil
 }
 
+func (index *OCI1Index) EditInstances(updates []ListEdit) error {
```

Comment:
It would be useful for `UpdateInstances` to delegate to the new more general `EditInstances`, so that we only have one copy of the “business logic”.

But let’s only add that to this PR only after the design of `EditInstances` settles.